pkgs/development/libraries/fastpbkdf2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation {
+  name = "fastpbkdf2-1.0.0";
+  
+  src = fetchurl {
+    url = https://github.com/ctz/fastpbkdf2/archive/v1.0.0.tar.gz;
+    sha256 = "492874e2088dd38cfa74b1b3a99c0b907ecb8da96c656e618353a2e825b8c152";
+  };
+  
+  buildInputs = [ openssl ];
+
+  preBuild = ''
+    makeFlagsArray=(CFLAGS="-std=c99 -O3 -g")
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{lib,include/fastpbkdf2}
+    cp *.a $out/lib
```

Comment:
Do they only ship static libraries? If there is some way to tweak build system to emit dynamic libraries (I think in case of macOS it's dylibs), it would be better.

Reply:
This package only has a static library, and I do not know of a way to make it generate a shared library without patching the Makefile.

Reply:
I see, OK!